CONTRIBUTING.md

```diff
@@ -125,10 +114,10 @@ You can also [reach us by email][contact].
 [github-flow]: https://guides.github.com/introduction/flow/
 [github-join]: https://github.com/join
 [how-contribute]: https://egghead.io/courses/how-to-contribute-to-an-open-source-project-on-github
-[issues]: https://guides.github.com/features/issues/
-[repo]: https://github.com/carpentries/maintainer-onboarding/
-[repo-issues]: https://github.com/carpentries/maintainer-onboarding/issues/
+[issues]: https://carpentries.org/help-wanted-issues/
```

Comment:
In the text, it isn't clear to me that "submit bug reports" would take me to the list of help-wanted issues. It used to go to GitHub help on issues. Is this intentional?